src/xdocs/config_filters.xml

```diff
@@ -986,5 +986,275 @@ public class UserService {
+                      <td><code>"// CHECKSTYLE:OFF"</code></td>
+                      <td>8.4</td>
+                  </tr>
+                  <tr>
+                      <td>onCommentFormat</td>
+                      <td>comment pattern to trigger filter to end suppression</td>
+                      <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                      <td><code>"// CHECKSTYLE:ON"</code></td>
+                      <td>8.4</td>
+                  </tr>
+                  <tr>
+                      <td>checkFormat</td>
+                      <td>check pattern to suppress</td>
+                      <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                      <td><code>".*"</code></td>
+                      <td>8.4</td>
+                  </tr>
+                  <tr>
+                      <td>messageFormat</td>
+                      <td>message pattern to suppress</td>
+                      <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                      <td><code>null</code></td>
+                      <td>8.5</td>
+                  </tr>
+              </table>
+              <p>
+                  offCommentFormat and onCommentFormat must have equal <a
+                      href="https://docs.oracle.com/javase/7/docs/api/java/util/regex/Matcher.html#groupCount()">paren counts</a>.
+              </p>
+          </subsection>
+          <subsection name="Examples">
+              <p>
+                  To configure a filter to suppress audit events between a comment
+                  containing <code>CHECKSTYLE:OFF</code> and a comment containing
+                  <code>CHECKSTYLE:ON</code>:
+              </p>
+              <source>
+                  &lt;module name=&quot;Checker&quot;&gt;
+                  ...
+                  &lt;module name=&quot;SuppressWithPlainTextCommentFilter&quot;/&gt;
+                  ...
+                  &lt;/module&gt;
+              </source>
+              <p>
+                  To configure a filter to suppress audit events between a comment
+                  containing line <code>BEGIN GENERATED CODE</code> and a comment
+                  containing line <code>END GENERATED CODE</code>:
+              </p>
+              <source>
+                  &lt;module name=&quot;Checker&quot;&gt;
+                  &lt;module name=&quot;SuppressWithPlainTextCommentFilter&quot;&gt;
+                  &lt;property name=&quot;offCommentFormat&quot; value=&quot;BEGIN GENERATED CODE&quot;/&gt;
+                  &lt;property name=&quot;onCommentFormat&quot; value=&quot;END GENERATED CODE&quot;/&gt;
+                  &lt;/module>
+                  &lt;/module&gt;
+              </source>
+              <source>
+                  //BEGIN GENERATED CODE
+                  @Override
+                  public boolean equals(Object obj) { ... } // No violation events will be reported
```

Comment:
I know these examples were copied/paste, but should we change these to non-java sources so it is clear this filter isn't recommended for Java code anymore?

Reply:
Done. In addition, I added [the warning](https://github.com/MEZk/checkstyle/blob/c3af06d8ed19f797d02b910fbb7926a0c1b790fd/src/xdocs/config_filters.xml#L1003) for the users.